src/lib/types/index.js

```diff
@@ -8,9 +8,8 @@ export type Triggers= {
 }
 
 export type InputFieldParams = {
-    captureField: Function,
-    fieldName: string,
-    triggers: Triggers
+  onCaptureInput: Function,
```

Comment:
Just to note: 
We know this is a function. We know what it is expecting, what it will return. Although flow likes this; it is better to use a full function typing, where we can, so that it clearly shows what it is doing. 
Ex:
```
  onCaptureInput: (id: string, value: string) => void // A dispatcher action will always be a void return because we don't expect anything back due to it going to the reducer and later pulling it out of the state.
```